src/main/java/org/apache/commons/lang3/RandomUtils.java

```diff
@@ -30,8 +30,10 @@
  * a better choice for applications with more stringent requirements
  * (performance and/or correctness).</p>
  *
+ * @deprecated for more optimized implementation from commons-rng
```

Comment:
Please add a URL to the Javadoc for the favored implementation.

Reply:
Sorry, I don't understand. What should I do?

Reply:
Added reference